lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -101,4 +104,36 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
+  if (!isChrome) {
+    it('should respect ignoreSearch when matching', function(done) {
+      var cacheName = 'test';
+      var options = {
+        ignoreSearch: true,
+        ignoreMethod: false,
+        ignoreVary: false,
+        prefixMatch: false,
+        cacheName: cacheName
+      };
+      var offline = new SimpleOfflineCache(cacheName, options);
+      var request = new Request('/index.html?param=zzz');
+      var response = new Response('contents');
+      caches.open(cacheName)
+      .then(function (cache) {
+        return cache.put(request.clone(), response.clone());
+      })
+      .then(function() {
+        var searchRequest = new Request('/index.html?param=xyz');
+        return offline.onFetch(searchRequest, null);
+      })
+      .then(function(response) {
+        expect(response).to.be.ok;
+        expect(response.text()).to.equal('contents');
```

Comment:
[`.text()` is a promise](https://developer.mozilla.org/en-US/docs/Web/API/Body/text), my friend.